roles/edpm_neutron_metadata/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+# Copyright 2021 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+- name: Include host prep tasks
+  ansible.builtin.import_tasks: install.yml
+
+- name: Include bootstrap tasks
+  ansible.builtin.import_tasks: bootstrap.yml
+
+- name: Configure OVS
```

Comment:
I wonder if OVS business belongs in this role at all. Though I don't know how dependencies between roles are managed. I think a good analogy here would be nova vs libvirt. There, nova pulls libvirt role as in:

```
roles/edpm_nova/molecule/default/prepare.yml:        name: "edpm_libvirt"
```

Reply:
this is not configure ovs but configure ovn-metadata-agent, it was a leftover (copy&paste typo)